### PR TITLE
Update what's new for 0.18

### PR DESCRIPTION
**Pull request by QuLogic**
A couple more changes that people did, plus typo fixes. Also add note about @greglucas as a dev.

### CODE REVIEW

File: docs/source/whats_new.rst [hunk 1]
```diff
@@ -10,16 +10,22 @@ For a full list of included Pull Requests and closed Issues, please see the
 Features
 --------
 
-* Kevin Donkers and Phil Elson made the AdaptiveScalar to the Natural
-  Earth Features the default. This will make the default features look
-  much nicer when plotting on zoomed in axes. (:pull:`1105`)
+* We are very pleased to announce that Greg Lucas has been added to the cartopy
```
Comment (greglucas) [inline docs/source/whats_new.rst:13]: Should `Cartopy` be capitalized? I see below it is actually listed as `CartoPy`.

File: docs/source/whats_new.rst [hunk 2]
```diff
@@ -10,16 +10,22 @@ For a full list of included Pull Requests and closed Issues, please see the
...
+  core development team. Greg (@greglucas) added the NightShade feature in the
```
Comment (greglucas) [inline docs/source/whats_new.rst:14]: I'm now a 'maintainer', you're on the core development team I think. I'm not sure the qualifiers on everything though. But, perhaps put your new upgrade in as well?

File: docs/source/whats_new.rst (hunk 1 above)
Comment (QuLogic) [reply docs/source/whats_new.rst:13]: I do prefer to capitalize, but we aren't particularly consistent about it, it seems.

File: docs/source/whats_new.rst (hunk 2 above)
Comment (QuLogic) [reply docs/source/whats_new.rst:14]: I just copied the existing ones; it said the same thing when I was added. I don't think there are any announcements for maintainer->core dev I could copy.

**Comment by QuLogic** [comment]
Please leave a comment if you want more changes (or open a PR), but I'll merge to tag the next beta.

**Pull request merged**